src/main/kotlin/org/oooc/kry/board/domain/entity/Comment.kt

```diff
@@ -41,7 +41,7 @@ class Comment (
     /* for bidirectional mapping */
 
     @OneToMany(mappedBy = "comment")
-    val commentVotes: List<CommentVote> = emptyList()
+    val commentVotes: MutableList<CommentVote> = mutableListOf()

```

Comment:
```suggestion
    val commentVotes: MutableList<CommentVote> = emptyList()
```
타입을 `MutableList<CommentVote>`라고 명시했기 때문에 `emptyList()`라고 써도 됩니다.